v19.2/performance.md

```diff
@@ -46,18 +46,18 @@ CockroachDB has **no theoretical scaling limit** and, in practice, can achieve n
 
 This chart shows that adding nodes increases throughput linearly while holding p50 and p99 latency constant. The concurrency for each scale was chosen to optimize throughput while maintaining an acceptable latency and can be observed in the table below.
 
-| Number of Nodes | Workers | Splits | Concurrency
-|-----------------|---------|--------|------------
-|       16        |    2    |  128   |     512
-|       32        |    4    |  256   |     512
-|       64        |    4    |  256   |     1024
-|       128       |    8    |  512   |     1024
-|       256       |    8    |  512   |     2048
+| Number of Nodes | Workers | Concurrency
```

Comment:
I feel like you may as well just multiply it out for people and not show the number of worker nodes? Why make people for the point (which is that we scaled concurrency linearly).

Reply:
Or perhaps add a 4th column?

Reply:
@nvanbenschoten said to remove it so the final version i have now is node, workers, concurrency